### ISSUE TITLE
Improve 1C positive auctions

### ISSUE THREAD

**Issue by bhamrick** [bhamrick/precision_system#40]
I am unhappy with our current 1C structure for a couple reasons:

* Responder shows shape earlier, and it's therefore often difficult for them to be captain. Example: 1C-1NT (8-11 or 14+ bal)-2S (nat)-3S (support)-3NT (SRAAACA, 16-18). Responder said they're balanced so opener knows whether 3NT or 4S is likely better, but responder doesn't.
* Controls (A=2, K=1) is generally a more useful metric for slam seeking than high card points, especially combined with keycard. Many times slam is likely if partner will have a king to fill in my minor suit, but if they have random queens and jacks it's not so useful.
* Generally, opener has more reliable source of tricks and wants to know if responder has honors to fill them in, so opener is typically more suited to be captain.

One (possibly somewhat outdated) style is for opener's non-jump new suit to be an asking bid with step responses:
1. No support (often defined as worse than Qxx), 0-3 controls
2. No support, 4+ controls
3. Qxx+ support, 0-3 controls
4. Qxx+ support, 4+ controls
5. Qxxx+ support, 4+ controls

If opener jumps to a suit that would be unilaterally setting trump and asking some question, and we would also need to establish a way to check an alternate suit after an initial negative support response.

Some more information: https://www.bridgehands.com/A/Alpha_Cuebid.htm

**Comment by shewu** [comment]
> One (possibly somewhat outdated) style is for opener's non-jump new suit to be an asking bid with step responses:

Is this after a fit is agreed? How will this work when responder is captain?

> Controls (A=2, K=1) is generally a more useful metric for slam seeking than high card points

If we choose to use a control ask, will SRAAACA be deprecated?

**Comment by bhamrick** [comment]
> > One (possibly somewhat outdated) style is for opener's non-jump new suit to be an asking bid with step responses:
> 
> Is this after a fit is agreed? How will this work when responder is captain?

This is right after the positive response e.g. 1C-1S, 2H is an ask about heart support. The point is that responder won't be captain.
> 
> > Controls (A=2, K=1) is generally a more useful metric for slam seeking than high card points
> 
> If we choose to use a control ask, will SRAAACA be deprecated?

Yes

**Comment by shewu** [comment]
> This is right after the positive response e.g. 1C-1S, 2H is an ask about heart support. The point is that responder won't be captain.

Then how do we agree on a fit? I assume that the 2H ask does not agree spades as the trump suit.
